data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic.container.template

```diff
@@ -0,0 +1,38 @@
+[Unit]
+Description=Ironic baremetal deployment service
+BindsTo=ironic-volume.service
+Requires=build-ironic-env.service wait-iptables-init.service image-customization.service
+{{ if .PlatformData.BareMetal.ProvisioningIP }}
+Requires=provisioning-interface.service
+{{ end }}
+{{ if .PlatformData.BareMetal.ProvisioningDNSMasq }}
+Wants=ironic-dnsmasq.service
+{{ end }}
+Wants=network-online.target ironic-httpd.service ironic-inspector.service ironic-ramdisk-logs.service
+After=network-online.target ironic-volume.service build-ironic-env.service wait-iptables-init.service provisioning-interface.service ironic-dnsmasq.service image-customization.service
+
+[Container]
+ContainerName=ironic
+Image=$IRONIC_IMAGE
+Exec=/bin/runironic
+Network=host
+PodmanArgs=--privileged
```

Comment:
Mental note: get rid of this, only dnsmasq needs to be privileged

Reply:
I've now removed all the --privileged args.